frontend/src/App.js

```diff
@@ -38,6 +41,9 @@ function App() {
     events: true,
     team: true,
     mentors: true,
+    mentorInterns: true,
```

Comment:
```suggestion
    mentorInterns: {},
```
Use `canFetch.mentorInterns.mentorId`